runtime/syntax/help_it.vim

```diff
@@ -0,0 +1,17 @@
+" Vim syntax file
+" Language:	Italian Vim program help files *.itx
+" Maintainer:	The Vim Project <https://github.com/vim/vim>
+" Last Change:	16 Oct 2024
+"
+" This script is sourced from syntax/help.vim.
```

Comment:
This doesn't seem to be the case however. Did you forget a separate `:runtime! syntax/help_it.vim` inside `syntax/help.vim`?

Reply:
It is sourced at here:
https://github.com/vim/vim/blob/c0cba184f54e052cf8f04baca24bdee72a4b7de6/runtime/syntax/help.vim#L153-L157

`syntax/help_ru.vim` is sourced by this as well.

Reply:
ah thanks, I missed that